addons/website_links/controller/main.py

```diff
@@ -12,7 +12,7 @@ class WebsiteUrl(http.Controller):
     def create_shorten_url(self, **post):
         if 'url' not in post or post['url'] == '':
             return {'error': 'empty_url'}
-        return request.env['link.tracker'].create(post).read()
+        return request.env['link.tracker'].with_context(website_id=request.website_routing).create(post).read()
```

Comment:
request.website_routing is not really appropriated. 
While it will works, you should probably work with request.env['website'].get_current_website()

Reply:
Where is that context key actually used? I dont think the website_links has code taking this key into consideration

Reply:
It's a typo from an earlier version of the PR, used for `get_current_website()`, I just removed it